documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -69,30 +110,71 @@ value.converter.apicurio.registry.url=http://apicurio:8080/api
+In your environment, you might want to use a provided {prodname} container to deploy {prodname} connectors that use Avro serializaion. Follow the procedure here to do that. In this procedure, you build a custom Kafka Connect container image for {prodname}, which uses the Avro converter. 
 
-[source,subs="+attributes"]
+.Prerequisites
+
+* You have the required permissions on a Kafka cluster.
+* You downloaded the {prodname} connector plug-in that you want to deploy with Avro serialization.
+
+.Procedure
+
+. Deploy an instance of {registry}. 
++
+ifdef::community[]
+The following example uses a non-production, in-memory, {registry}  instance:
++
+[source,subs="attributes+"]
 ----
 docker run -it --rm --name apicurio \
     -p 8080:8080 apicurio/apicurio-registry-mem:{apicurio-version}
 ----
+endif::community[]
+ifdef::product[]
+Follow the instructions in 
+{LinkServiceRegistryGetStart}#installing-registry-operatorhub[NameServiceRegistryGetStart, Installing Service Registry from the OpenShift OperatorHub].
+endif::product[]
+
+. Build a {prodname} container image that contains the Avro converter:
++
+ifdef::community[]
+.. Copy link:https://github.com/debezium/debezium-examples/blob/master/tutorial/debezium-with-apicurio/Dockerfile[`Dockerfile`] to a convenient location. This file has the following content: 
++
+[listing,subs="attributes",options="nowrap"]
+----
+ARG DEBEZIUM_VERSION
+FROM debezium/connect:$DEBEZIUM_VERSION
+ENV KAFKA_CONNECT_DEBEZIUM_DIR=$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-mysql
+ENV APICURIO_VERSION={apicurio-version}
 
-Build a {prodname} image with Avro converter from https://github.com/debezium/debezium-examples/blob/master/tutorial/debezium-with-apicurio/Dockerfile[Dockerfile]:
+RUN cd $KAFKA_CONNECT_DEBEZIUM_DIR &&\
+    curl https://repo1.maven.org/maven2/io/apicurio/apicurio-registry-distro-connect-converter/$APICURIO_VERSION/apicurio-registry-distro-connect-converter-$APICURIO_VERSION-converter.tar.gz | tar xzv
+----
 
-[source]
-[subs="attributes"]
+.. Run the following command: 
++
+[source,subs="attributes+"]
 ----
 docker build --build-arg DEBEZIUM_VERSION={debezium-docker-label} -t debezium/connect-apicurio:{debezium-docker-label} .
 ----
-
-Run a Kafka Connect image configured to use Avro:
-
-[source]
-[subs="attributes"]
+endif::community[]
+ifdef::product[]
+.. Download the link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=red.hat.integration&downloadType=distributions[{registry} Kafka Connect] zip file. 
+.. Extract the content into the directory that contains the {prodname} connector that you are configuring to use Avro serialization. 
+.. Create a custom image for Kafka Connect. See link:{LinkCDCInstallOpenShift}[{NameCDCInstallOpenShift}, Creating a container image from the Kafka Connect base image] for an example of how to do this. Start with the `Dockerfile` in that example. Then add the {registry} converters to the connector directories. 
+endif::product[]
```

Comment:
This section talks about Docker file, which means building a custom Kafka Connect image (which is correct), the issue comes in my next comment